bigtable-dataflow-parent/bigtable-beam-import/src/main/java/com/google/cloud/bigtable/beam/TemplateUtils.java

```diff
@@ -50,11 +50,21 @@
 public class TemplateUtils {
   /** Builds CloudBigtableTableConfiguration from input runtime parameters for import job. */
   public static CloudBigtableTableConfiguration BuildImportConfig(ImportOptions opts) {
+    return BuildImportConfigWithCustomUserAgent(opts, "ImportJob");
+  }
```

Comment:
I dont think you need to maintain the old method...all of this is marked as internal api

Reply:
Will remove this in the follow up PR. There are bots that were enforcing public API for classes. I will disable them for this class and remove this method.